Documentation/spelling_wordlist.txt

```diff
@@ -215,17 +219,24 @@ backporting
 backports
 balancer
 balancers
+bandwidthManager
 battlestation
 bcc
 behaviour
 benchmarking
 benefitting
 bgpd
+binPath
+bindProtection
 blog
 bnxt
 bookinfo
 boolean
+bootstrapFile
 bpf
+bpfClockProbe
+bpfMasquerade
```

Comment:
```suggestion
```

Reply:
this is still required as the flag is included in prior version upgrade documentation